.travis.yml

```
@@ -103,3 +103,15 @@ matrix:
     - <<: *cmake-build-test
       name: "CMake Azure IoT Hub example - debug (DISCO_L475VG_IOT01A)"
       env: NAME=cmake_test TARGET_NAME=DISCO_L475VG_IOT01A PROFILE=debug CACHE_NAME=debug-DISCO_L475VG_IOT01A
+
+      - <<: *cmake-build-test
```

```suggestion
    - <<: *cmake-build-test
```
Indentation. Travis wasn't able to parse this file.